examples/build_image.sh

```diff
@@ -22,6 +22,8 @@ IMAGE_TAG=${IMAGE_TAG:-v0.3.0}
 EXAMPLE_REPO_PREFIX=${IMAGE_REPO}/sedna-example-
 
 dockerfiles=(
+federated-learning-mistnet-client.Dockerfile
+federated-learning-mistnet.Dockerfile
```

Comment:
I suggest it's better to name it "federated-learning-mistnet-aggregation".

Reply:
updated